plugins/scss/index.js

```diff
@@ -156,9 +156,12 @@ const addScssRequires = function(pack, dir) {
 
 const writer = function(thisPackage, outputDir) {
   if (scssPaths.length || scssEntries.length) {
-    var options = {
-      'include-path': scssPaths
-    };
+    var options = {};
+    if (thisPackage.build.scssCompiler === 'sass') {
```

Comment:
Instead of requiring this information in two places, try detecting it from the dependencies directly.